src/components/Admin/AdminHashtags/AdminHashtagsPagination.tsx

```diff
@@ -0,0 +1,32 @@
+import { Pagination } from 'hds-react';
+import { FC, MouseEvent, useCallback } from 'react';
+
+interface IAdminHashtagsPaginationProps {
+  pageCount: number;
+  page: number;
+  onPageChange: (event: MouseEvent<HTMLButtonElement | HTMLAnchorElement>, page: number) => void;
+}
+
+const AdminHashtagsPagination: FC<IAdminHashtagsPaginationProps> = ({
+  pageCount,
+  page,
+  onPageChange,
+}) => {
+  const pageHref = useCallback(() => '#', []);
```

Comment:
Is this supposed to be function or just value of #?

Reply:
This is just a memoized function that the HDS needs Pagination https://hds.hel.fi/components/pagination/code/. 

Trying to return a variable gives this error:
`Type 'string' is not assignable to type '(index: number) => string'.`